config/recipes/beats/metricbeat_openshift.yaml

```diff
@@ -0,0 +1,222 @@
+apiVersion: beat.k8s.elastic.co/v1beta1
+kind: Beat
+metadata:
+  name: metricbeat
+  namespace: openshift-monitoring
+spec:
+  type: metricbeat
+  version: 7.10.0
+  elasticsearchRef:
```

Comment:
I think we expect for the dependencies (Elasticsearch and Kibana) to be included in the recipe.

Reply:
Added resources for both ES and Kibana